apps/bcc3d.ca/pages/donations.html

```diff
@@ -9,7 +9,7 @@
   <h2>Donations</h2>
 
   <div class="row">
-    <div class="col-md-4 order-2">
+    <div class="col-md-4 order-2" style="max-height: 80vh;">
```

Comment:
The `max-height` attribute was put in place so if the donors list is too long, the page doesn't get longer, but instead uses a scrollbar to display donors.